src/ttsim/interface_dag_elements/shared.py

```diff
@@ -30,6 +32,29 @@ def to_datetime(date: datetime.date | DashedISOString) -> datetime.date:
         )
 
 
+def reorder_ids(ids: np.ndarray) -> np.ndarray:
+    """Make ID's consecutively numbered.
+
+    Takes the given IDs and replaces them by consecutive numbers
+    starting at 0.
+
+    [43,44,70,50] -> [0,1,3,2]
+
+    """
+
+    sorting = np.argsort(ids)
+    ids_sorted = ids[sorting]
+    index_after_sort = np.arange(ids.shape[0])[sorting]
+
+    # Look for difference from previous entry in sorted array
+    diff_to_prev = np.where(np.diff(ids_sorted) >= 1, 1, 0)
+
+    # Sum up all differences to get new id
+    cons_ids = np.concatenate((np.asarray([0]), np.cumsum(diff_to_prev)))
```

Comment:
```suggestion
    consecutive_ids = np.concatenate((np.asarray([0]), np.cumsum(diff_to_prev)))
```
`cons` is hard-wired to mean `consumption` in my old brain...